projects.json

```diff
@@ -308,6 +308,30 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/maxhumber/BreadBuddy",
+    "path": "BreadBuddy",
+    "branch": "master",
+    "maintainer": "max@bracket.ltd",
+    "compatibility": {
+      "5.6.1": {
```

Comment:
Can we update this to 5.0? we are trying to keep the compatibility to supported swift versions. 
* Swift 4.0
* Swift 4.2
* Swift 5.0

Reply:
Yeah, shouldn’t be a problem. Would you like me to adjust? Or leave it to you?